tools/camera/groupFWHM.xml

```diff
@@ -0,0 +1,120 @@
+<tool id="CAMERA_groupFWHM" name="CAMERA.groupFWHM" version="1.0.0+camera@TOOL_VERSION@-galaxy@VERSION_SUFFIX@">
+	<description>Group peaks of a xsAnnotate object according to their retention time into pseudospectra-groups.</description>
+
+	<macros>
+		<import>macros.xml</import>
+	</macros> 
+
+	<expand macro="requirements"/>
+	<expand macro="stdio"/>
+
+	<command detect_errors="exit_code"><![CDATA[
+		@COMMAND_RSCRIPT@/CAMERA_groupFWHM.R 
+		'$image' 
+		$groupfwhm.sigma 
+		$groupfwhm.perfwhm 
+		$groupfwhm.intval
+		$groupfwhm.convertRTMinute
+		$groupfwhm.numDigitsMZ
+		$groupfwhm.numDigitsRT
+		]]>
+	</command>
+
+	<inputs>
+		<param name="image" type="data" label="RData file" format="rdata.xcms.fillpeaks,rdata" help="output file from another function xcms (fillPeaks)" />
```

Comment:
```suggestion
		<param name="image" type="data" label="RData file" format="rdata.xcms.fillpeaks" help="output file from another function xcms (fillPeaks)" />
```

Reply:
I've kept the rdata type rather than the rdata.xcms.fillpeaks for greater flexibility when depositing our data.